make/mt-os/mt-stm32f4.mk

```diff
@@ -11,12 +11,16 @@ XI ?= $(XI_BINDIR)/libxively.a
 include make/mt-utils/mt-get-gnu-arm-toolchain.mk
 
 XI_STM32_PATH_SDK = $(HOME)/Downloads/xively-client-artifactory/st/STM32Cube_FW_F4_V1.13.0
+XI_STM32_PATH_SDK_ST = $(HOME)/Downloads/STM32F4xx_DSP_StdPeriph_Lib_V1.8.0/Libraries
 
 #####################
 # LWIP configurations
 #####################
 XI_STM32_PATH_SDK_LWIP = $(XI_STM32_PATH_SDK)/Middlewares/Third_Party/LwIP
 
+XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK_ST)/STM32F4xx_StdPeriph_Driver/inc
```

Comment:
do we need all these include paths? I see only a single file included in BSP RNG STM: stm32f4xx_rng.h, probably it is located at a single place and not quantum-existance in all three locations.
Sorry maybe there are more includes which require all paths, just trying to preventing adding unnecessary include paths.